go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/metronlab/bow
 go 1.13
 
 require (
-	github.com/apache/arrow/go/arrow v0.0.0-20200727110821-c49e4ca157f6
```

Comment:
could you add a comment that reference the github repo with taken commit or release for traceability ?